recipe/conda_build_config.yaml

```diff
@@ -44,6 +44,12 @@ cuda_compiler_version:
   - 10.0                       # [linux64]
   - 10.1                       # [linux64]
   - 10.2                       # [linux64]
+cudnn:
+  - undefined
```

Comment:
I think this needs to be `None`?

Reply:
It doesn't matter what this value is

Reply:
Doesn't matter to conda sure, it may matter to the bot however. I need to dig more.

Reply:
I can change it to None if that helps

Reply:
Thank you! I think I put in a more general fix that appears to be working.

Reply:
So no need.